Patch to set the correct paths for LAMMPS dependencies
Thekla Loizou July 2015
--- src/MAKE/Makefile.openmpi.orig	2014-10-17 12:46:28.862970000 +0300
+++ src/MAKE/Makefile.openmpi	2014-10-17 12:47:01.973616000 +0300
@@ -38,9 +38,9 @@
 # PATH = path for MPI library
 # LIB = name of MPI library
 
-MPI_INC =       
-MPI_PATH = 
-MPI_LIB =	
+MPI_INC =       -I$(EBROOTOPENMPI)/include
+MPI_PATH =      -L$(EBROOTOPENMPI)/lib
+MPI_LIB =       -lmpi -lpthread
 
 # FFT library, OPTIONAL
 # see discussion in doc/Section_start.html#2_2 (step 6)
@@ -49,9 +49,9 @@
 # PATH = path for FFT library
 # LIB = name of FFT library
 
-FFT_INC =       -DFFT_FFTW
-FFT_PATH = 
-FFT_LIB =	-lfftw
+FFT_INC =       -DFFT_FFTW3
+FFT_PATH =      -L$(EBROOTFFTW)/lib
+FFT_LIB =       -lfftw3
 
 # JPEG and/or PNG library, OPTIONAL
 # see discussion in doc/Section_start.html#2_2 (step 7)
